docs/marker.md

```diff
@@ -62,3 +62,16 @@ type Point {
   y: Number,
 }
 ```
+
+## Children Components
+
+Children components can be added within a Marker and rendered content will replace the marker symbol.  This is a way of creating custom markers and allowing use of native SVGs.
```

Comment:
Maybe try to be consistent between the explanation and the example? 
Or make the explanation more broad?

Just my 2c